osler/workup/tests/test_scripts.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+from osler.workup.models import Workup
+from osler.workup.scripts import workup_date_checker
+from .factories import WorkupFactory
+import datetime
+
+class testScripts(TestCase):
```

Comment:
This class name sounds generic, also the first letter should be capitalized. Something like `TestUtilityScripts` could work